nkms/characters.py

```diff
@@ -367,19 +357,15 @@ def alice(self):
 
     @alice.setter
     def alice(self, alice_object):
-        self.learn_about_actor(alice_object)
         self._alice = alice_object
 
     def follow_treasure_map(self, hrac):
         for ursula_interface_id in self.treasure_maps[hrac]:
             # TODO: perform this part concurrently.
-            getter = self.server.get(ursula_interface_id)
-            loop = asyncio.get_event_loop()
-            value = loop.run_until_complete(getter)
-            
+            value = self.server.get_now(ursula_interface_id)
+
             # TODO: Make this much prettier
-            header, signature, ursula_pubkey_sig, _hrac, (port, interface, ttl) =\
-            dht_value_splitter(value, msgpack_remainder=True)
+            header, signature, ursula_pubkey_sig, _hrac, (port, interface, ttl) = dht_value_splitter(value, msgpack_remainder=True)
```

Comment:
I still think we need to make this a `CryptoKit` of some sort.

Reply:
haha yeah.  No question.  That's what I think every time I run into that TODO.